extensions/libraries/redcore/html/rformbehavior.php

```diff
@@ -19,23 +19,18 @@
 abstract class JHtmlRformbehavior extends JHtmlFormbehavior
 {
 	/**
-	 * Array containing information for loaded files
-	 *
-	 * @var  array
-	 */
-	protected static $loaded = array();
-
-	/**
 	 * Method to load the AJAX Chosen library
 	 *
 	 * If debugging mode is on an uncompressed version of AJAX Chosen is included for easier debugging.
 	 *
-	 * @param   JRegistry  $options  Options in a JRegistry object
-	 * @param   mixed      $debug    Is debugging mode on? [optional]
+	 * @param   Registry  $options  Options in a Registry object
+	 * @param   mixed     $debug    Is debugging mode on? [optional]
 	 *
 	 * @return  void
+	 *
+	 * @since   3.0
 	 */
-	public static function ajaxchosen(JRegistry $options, $debug = null)
+	public static function ajaxchosen(Registry $options, $debug = null)
```

Comment:
Isn't there a conflict with Registry and 2.5?  I recall we had that some time ago